ironic_tempest_plugin/tests/scenario/ironic_standalone/test_basic_ops.py

```diff
@@ -437,17 +437,16 @@ class BaremetalRedfishIPxeWholediskHttpLink(
     def test_ip_access_to_server(self):
         self.boot_and_verify_node()
 
-class BaremetalIdracDirectWholediskHttpLink(
+
+class BaremetalIdracVirtualMedia(
         bsm.BaremetalStandaloneScenerioTest):
 
+    mandatory_attr = ['driver', 'boot_interface']
     api_microversion = '1.31'
```

Comment:
Add a credentials parameter. Also Add Dell copyright. Refer to BIOS/Management test case in upstream.
https://review.opendev.org/c/openstack/ironic-tempest-plugin/+/829665/2/ironic_tempest_plugin/tests/scenario/ironic_standalone/test_bios.py#64

Reply:
done